util/util.go

```diff
@@ -350,3 +354,73 @@ func WriteWireGuardServerConfig(tmplBox *rice.Box, serverConfig model.Server, cl
 
 	return nil
 }
+
+// RestartWireGuardProces to restart wireguard interface when a new client is added or deleted
+func RestartWireGuardProces(globalSettings model.GlobalSetting) error {
+	ss := strings.Split(globalSettings.ConfigFilePath, "/")
+	wireguardInterfaceName := strings.Split(ss[len(ss)-1], ".")[0]
+	app := "wg-quick"
+
+	cmd := exec.Command(app, "up", wireguardInterfaceName)
+	_, err := cmd.Output()
+
+	if err != nil {
+		return err
+	}
+
+	cmd = exec.Command(app, "down", wireguardInterfaceName)
+	_, err = cmd.Output()
+
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// SendMailToNewUser Send mail to newly created user
+func SendMailToNewUser(client *model.Client) error {
+	m := mail.NewV3Mail()
+
+	from := mail.NewEmail("Wireguard", SendGridUser)
```

Comment:
Where is setup SendGrid? Is there a dedicated configuration for Wireguard? If yes is it setup with IAC?